### PR TITLE
[plugin.video.fatstone] 1.0.0

### PR DESCRIPTION
**Pull request by haakonnessjoen**

### CODE REVIEW

File: plugin.video.fatstone/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.fatstone"
+       name="Fatstone.TV"
+       version="1.0.0"
+       provider-name="trippel-m">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.requests" version="1.0.4"/>
```
Comment (learningit) [inline plugin.video.fatstone/addon.xml]: If you intend to submit this for Jarvis and versions later, please set xbmc.python version="2.24.0"
Please see http://kodi.wiki/view/Addon.xml#Dependency_versions for other version values, if you want something other than Jarvis

**Comment by learningit** [comment]
Please remove the .gitignore and .gitattributes files in the commit.

**Comment by learningit** [comment]
The rest looks OK.

**Comment by haakonnessjoen** [comment]
I have sent a new pull request, with the requested changes.

**Comment by Razzeee** [comment]
Just for your information, you can update pull requests, by pushing or force pushing to the same branch your PR is based on.

**Comment by haakonnessjoen** [comment]
I had to change the branch, since I wanted to push against isengard instead of jarvis

**Pull request closed without merging**